pkg/template/servicebroker/apis/config/v1/types.go

```diff
@@ -0,0 +1,15 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TemplateServiceBrokerConfig holds information related to the template
+// service broker
+type TemplateServiceBrokerConfig struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// TemplateNamespaces indicates the namespace(s) in which the template service
+	// broker looks for templates to serve to the catalog.
+	TemplateNamespaces []string `json:"templateNamespaces"`
```

Comment:
(note that @jim-minter is already working on wiring the cluster up loglevel through to the TSB pod, presumably via the CLI)